parlai/core/params.py

```diff
@@ -638,6 +832,24 @@ def parse_known_args(self, args=None, namespace=None, nohelp=False):
             args = [a for a in args if a != '-h' and a != '--help']
         return super().parse_known_args(args, namespace)
 
+    def _load_known_opts(self, optfile, parsed):
+        with open(optfile, 'r', encoding='utf-8') as handle:
```

Comment:
can you call this existing utility function instead to allow for backwards compatibility with pickled opt files? https://github.com/facebookresearch/ParlAI/blob/9e1a417de7ca7346fd3bc2995bf6c847fb051c58/parlai/core/agents.py#L412